lua/examples/lpm_policy.lua

```diff
@@ -0,0 +1,187 @@
+local policylib = require("gatekeeper/policylib")
+local ffi = require("ffi")
```

Comment:
Let's avoid having multiple policy examples. Our intent is to eventually arrive at a policy example that is easy to adapt to one's needs and has meaningful protection, so to lower the bar to get started with Gatekeeper. Thus, combine the file `lua/examples/lpm_policy.lua` into `lua/examples/policy.lua`.

A good example for the integration of both policies is to have bogon files (see in #208 from where you can download examples) for IPv4 and IPv6. The policy loads these bogon files into the LPM tables and refuses any flow whose source is a bogon.